### PR TITLE
PYTHON-2718 Test redaction of security sensitive command monitoring events

### PR DESCRIPTION
**Pull request by prashantmital**
Ready for review. Failures are unrelated to the change.

### CODE REVIEW

**Comment by prashantmital** [comment]
@ShaneHarvey `monitoring._SENSITIVE_COMMANDS` is used in many places as a source of truth for what is a sensitive command (e.g. in `compression_support.py`, in `test_auth.py`). Should I create a new method that accepts command name + payload and returns a boolean if the command is sensitive to use in these places instead?

File: test/unified_format.py [hunk 1]
```diff
@@ -124,8 +124,16 @@ def is_run_on_requirement_satisfied(requirement):
             elif client_context.server_parameters[param] != val:
                 params_satisfied = False
 
+    auth = requirement.get('auth')
```
Comment (ShaneHarvey) [inline test/unified_format.py:127]: I already made this change. Please revert and rebase.

File: test/test_command_monitoring_spec.py [hunk 2]
```diff
@@ -239,5 +241,11 @@ def create_tests():
 
 create_tests()
 
+
+globals().update(generate_test_classes(
+    os.path.join(_TEST_PATH, 'unified'),
+    module=__name__,))
```
Comment (ShaneHarvey) [inline test/test_command_monitoring_spec.py:247]: Please move this to a new file `test/test_command_monitoring_unified.py` and add a corresponding `test/load_balancer/test_command_monitoring_unified.py`.

**Review by ShaneHarvey — COMMENTED** [review]
> Should I create a new method that accepts command name + payload and returns a boolean if the command is sensitive to use in these places instead?

The tests are all passing now correct? If so, I don't see a reason to make that change.

File: test/unified_format.py (hunk 1 above)
Comment (prashantmital) [reply test/unified_format.py]: 👍🏽

File: test/test_command_monitoring_spec.py (hunk 2 above)
Comment (prashantmital) [reply test/test_command_monitoring_spec.py]: Ah... Thanks for pointing this out. Done.

**Review by ShaneHarvey — APPROVED** [review]
LGTM!

**Pull request merged**